packages/library-base/src/commonMain/kotlin/io/realm/Configuration.kt

```diff
@@ -240,6 +264,24 @@ interface Configuration {
         fun encryptionKey(encryptionKey: ByteArray) =
             apply { this.encryptionKey = validateEncryptionKey(encryptionKey) } as S
 
+        /**
+         * Due to the way Realm allocates space on disk, it is sometimes the case that more space
+         * is allocated than what is actually needed, making the realm file larger than what it
+         * needs to be. This mostly occurs when writing larger binary blobs to the file.
+         *
+         * The space will be used by subsequent writes, but in the interim period the file will
+         * be larger than what is strictly needed.
+         *
+         * This method makes it possible to define a function that determines whether or not
+         * the file should be compacted when the realm is opened, optimizing how much disk size
+         * is used.
+         *
+         * @param callback The callback called when opening the realm file. The return value
```

Comment:
Maybe also describe the default argument in the docs. Or are this automatically references in Dokka?

Reply:
It is visible in Dokka, but there was no Link to it. I'll create an explicit link.